packages/delayedInbox-l2msg/scripts/normalTx.js

```diff
@@ -0,0 +1,175 @@
+const { providers, Wallet, ethers } = require('ethers')
+const hre = require('hardhat')
+const { arbLog, requireEnvVariables } = require('arb-shared-dependencies')
+const { getL2Network } = require("@arbitrum/sdk-nitro/dist/lib/dataEntities/networks")
+requireEnvVariables(['DEVNET_PRIVKEY', 'L2RPC', 'L1RPC'])
+
+/**
+ * Set up: instantiate L1 / L2 wallets connected to providers
+ */
+const walletPrivateKey = process.env.DEVNET_PRIVKEY
+
+const L2MSG_signedTx = 4
+
+const l1Provider = new providers.JsonRpcProvider(process.env.L1RPC)
+const l2Provider = new providers.JsonRpcProvider(process.env.L2RPC)
+
+const l1Wallet = new Wallet(walletPrivateKey, l1Provider)
+const l2Wallet = new Wallet(walletPrivateKey, l2Provider)
+
+const main = async () => {
+  await arbLog('DelayedInbox normal contract call (L2MSG_signedTx)')
+
+  const l2Network = await getL2Network(await l2Wallet.getChainId())
+
+  /**
+   * We deploy greeter to L2, to see if delayed inbox tx can be executed as we thought
+   */
+  const L2Greeter = await (
+    await hre.ethers.getContractFactory('Greeter')
+  ).connect(l2Wallet)
+
+  console.log('Deploying Greeter on L2 👋👋')
+
+  const l2Greeter = await L2Greeter.deploy('Hello world')
+  await l2Greeter.deployed()
+  console.log(`deployed to ${l2Greeter.address}`)
+
+  /**
+   * Let's log the L2 greeting string
+   */
+  const currentL2Greeting = await l2Greeter.greet()
+  console.log(`Current L2 greeting: "${currentL2Greeting}"`)
+
+  console.log(
+    `Now we send a l2 tx through l1 delayed inbox (Please don't send any tx on l2 using ${l2Wallet.address} during this time):`
+  )
+
+  /**
+   * Here we have a new greeting message that we want to set as the L2 greeting; we'll be setting it by sending it as a message from delayed inbox!!!
+   */
+  const newGreeting = 'Greeting from delayedInbox'
+
+  const GreeterIface = l2Greeter.interface
+
+  const calldatal2 = GreeterIface.encodeFunctionData('setGreeting', [
+    newGreeting,
+  ])
+
+  /**
+   * Encode the l2's signed tx so this tx can be executed on l2
+   */
+  const l2GasPrice = await l2Provider.getGasPrice()
+
+  const transactionl2Request = {
+    data: calldatal2,
+    to: l2Greeter.address,
+    nonce: await l2Wallet.getTransactionCount(),
+    value: 0,
+    gasPrice: l2GasPrice,
+    chainId: l2Wallet.chainId,
+    from: l2Wallet.address,
+  }
+
+  let l2GasLimit
+
+  try {
+    l2GasLimit = await l2Provider.estimateGas(transactionl2Request)
+  } catch (error) {
+    console.error(
+      "execution failed (estimate gas failed), try check your account's balance?"
+    )
```

Comment:
I think we want to print that then still throw
```suggestion
    console.error(
      "execution failed (estimate gas failed), try check your account's balance?"
    );
    throw error;
```